RSS Reader/RSS Reader/Categories/UIViewController/UIViewController+ViewControllerPresentable.m

```diff
@@ -0,0 +1,27 @@
+//
+//  UIViewController+AlertPresentable.m
+//  RSS Reader
+//
+//  Created by Lina Loyko on 11/27/20.
+//
+
+#import "UIViewController+ViewControllerPresentable.h"
+
+@implementation UIViewController (ViewControllerPresentable)
+
+- (void)showAlertWithTitle:(nonnull NSString *)title message:(nonnull NSString *)message {
```

Comment:
Please, move it into the extension of the controller, since you are going to use it frequently across the app.

Reply:
It’s already in the controller extension

Reply:
Great!